code/modules/mob/inventory.dm

```diff
@@ -291,22 +291,22 @@
 
 //for when you want the item to end up on the ground
 //will force move the item to the ground and call the turf's Entered
-/mob/proc/dropItemToGround(obj/item/I, force = FALSE, thrown = FALSE)
-	return doUnEquip(I, force, drop_location(), FALSE, was_thrown = thrown)
+/mob/proc/dropItemToGround(obj/item/I, force = FALSE, thrown = FALSE, bypass_strip_delay_other)//NSV13
+	return doUnEquip(I, force, drop_location(), FALSE, was_thrown = thrown, bypass_strip_delay_other = bypass_strip_delay_other)//NSV13
```

Comment:
```suggestion
/mob/proc/dropItemToGround(obj/item/I, force = FALSE, thrown = FALSE, bypass_strip_delay_other = TRUE)//NSV13
	return doUnEquip(I, force, drop_location(), FALSE, TRUE, thrown, bypass_strip_delay_other)//NSV13
```
This is a lot cleaner, also ditto regarding the var length and don't forget to set the default to TRUE or it'll be null/FALSE.

Reply:
Doing this removes the delay from stripping yourself, this is what I tried doing at first.

Reply:
I'm doing this partly because it's the only thing that worked(I've tried all kinds of shit😑), and partly because this is also how the already implemented bypass_equip_delay_self works. I didn't really wanna have to completely rework how inventory handling is done.